src/core/graph.js

```diff
@@ -169,80 +169,76 @@ export class Graph<NP, EP> {
   }
 
   /**
-   * Gets the array of all out-edges from the node at the given address.
-   * The order of the resulting array is unspecified.
+   * Find the neighborhood of a given nodeAddress
+   *
+   * By neighborhood, we mean every `{edge, neighbor}` such that edge
+   * has `nodeAddress` as either `src` or `dst`, and `neighbor` is the
+   * address at the other end of the edge.
```

Comment:
…and such that the filter is satisfied.

Reply:
Added a description of how the options work.